internal/pkg/template/templates/workloads/partials/cf/vpc-connector.yml

```diff
@@ -0,0 +1,39 @@
+ServiceSecurityGroup:
+  Type: AWS::EC2::SecurityGroup
+  Metadata:
+    'aws:copilot:description': 'A security group for your addons to allow ingress'
+  Properties:
+    GroupDescription: !Join [ '', [ !Ref AppName, '-', !Ref EnvironmentName, !Ref WorkloadName ] ]
+    VpcId:
+      Fn::ImportValue:
+        !Sub '${AppName}-${EnvName}-VpcId'
+    Tags:
+      - Key: Name
+        Value: !Sub 'copilot-${AppName}-${EnvironmentName}-${WorkloadName}'
+
+VpcConnector:
+  Type: AWS::AppRunner::VpcConnector
+  DependsOn: EnvControllerAction # Ensure NAT gateways are created before connecting to a VPC.
+  Metadata:
+    'aws:copilot:description': 'A VPC connector to link your service to private subnets in your environment'
+  Properties:
+    Subnets:
+      Fn::Split:
+        - ","
+        - Fn::ImportValue:
+            !Sub '${AppName}-${EnvName}-PrivateSubnets'
+    SecurityGroups:
+    - !Ref ServiceSecurityGroup
+    Tags:
+      - Key: copilot-application
+        Value: !Ref AppName
+      - Key: copilot-environment
+        Value: !Ref EnvName
+      - Key: copilot-service
+        Value: !Ref WorkloadName
+      {{- if .Tags }}
+      {{- range $name, $value := .Tags }}
+      - Key: {{$name}}
+        Value: {{$value}}
+      {{- end }}
+      {{- end }}
```

Comment:
Why do we need this 🤔

Reply:
By default app runner doesn't trickle down stack tags to the individual resources, so we need to redeclare them here.

We let them know and they're looking into fixing that AFAIK.

The `.Tags` is a special field that's available in rdws manifests: https://aws.github.io/copilot-cli/docs/manifest/rd-web-service/ although we are missing docs for it 🤦

Reply:
Yeah but
```yaml
    Tags:
      - Key: copilot-application
        Value: !Ref AppName
      - Key: copilot-environment
        Value: !Ref EnvName
      - Key: copilot-service
        Value: !Ref WorkloadName
```
should be handled by CFN no?

Reply:
> By default app runner doesn't trickle down stack tags to the individual resources, so we need to redeclare them here.

nope unfortunately not today !

Reply:
oooh gotcha

Reply:
Maybe add a comment describing the reason why this is an exception?